### PR TITLE
#162162996 Document API using swagger

### PR DESCRIPTION
**Pull request by okellogabrielinnocent**
### What does this PR do?
 Implements swagger documentation
#### Description of Task to be completed?
- Do a `pip install django-rest-swagger` on command.
- Add ` 'rest_framework_swagger',` on application setting.py file under the `INSTALLED_APPS`.
- In the url.py module,  import `get_swagger_view` from `rest_framework_swagger.views`.
- Then add `url(r'^swagger/', schema_view)` in the `urlpatterns`,

#### How should this be manually tested?
- Run the server.
- Navigate to  `http://localhost:8000/api/swagger/` in the browser.

#### What are the relevant pivotal tracker stories?
[-](#162162996)
#### Screenshots (if appropriate)
image

### CODE REVIEW

File: authors/settings.py [hunk 1]
```diff
@@ -85,7 +86,7 @@
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': os.environ.get('DB_NAME', 'ahdb'),
         'USER': os.environ.get('DB_USER', 'postgres'),
-        'PASSWORD': os.environ.get('PASSWORD', ''),
+        'PASSWORD': os.environ.get('PASSWORD', 'moschinogab19'),
```
Comment (billkabanga) [inline authors/settings.py:89]: I think as we had agreed the password for the local database config should be kept open.

**Comment by billkabanga** [comment]
Good work done. LGTM.

File: authors/settings.py (hunk 1 above)
Comment (malep2007) [reply authors/settings.py:89]: @billkabanga ,was this resolved?

Comment (arthurarty) [reply authors/settings.py:89]: No this was not resolved. We will have to fix it in this sprint.

Comment (okellogabrielinnocent) [reply authors/settings.py:89]: Guys let me resolve this on Monday morning, I forgot of that when we were pushing.

**Pull request merged**